.circleci/config.yml

```diff
@@ -125,7 +125,18 @@ commands:
             set -x
             oc cancel-build $(oc get bc --output=name)
             for bc in $(oc get bc --output=name); do oc start-build $bc --wait --follow & done
-            wait
+            echo "List of build image processes:"
+            for job in $(jobs -p); do
+              ps aux | grep $job
```

Comment:
I tried to execute:
```
ps -o cmd fp
```
That works locally on my machine but seems that CircleCI uses another shell or shell version so for the moment this would be enough.